src/gui/raster/qgsrasterlayerproperties.cpp

```diff
@@ -835,6 +835,14 @@ void QgsRasterLayerProperties::apply()
   if ( !mRasterLayer->isValid() )
     return;
 
+  // apply all plugin dialogs
+  const auto constMLayerPropertiesPages = mLayerPropertiesPages;
+  for ( QgsMapLayerConfigWidget *page : constMLayerPropertiesPages )
```

Comment:
Or better:

```suggestion
  for ( QgsMapLayerConfigWidget *page : std::as_const(mLayerPropertiesPages) )
```

Reply:
Ok, thanks. 
I don't know much about C++ and just copied these lines from qgsvectorlayerproperties.cpp